inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/core/impl/AuditServiceImpl.java

```diff
@@ -373,6 +376,7 @@ private List<AuditVO> doAggregate(List<AuditVO> auditVOList, String format) {
         for (AuditVO auditVO : auditVOList) {
             AuditVO statInfo = new AuditVO();
             ConcurrentHashMap<String, AtomicLong> countMap = new ConcurrentHashMap<>();
+            ConcurrentHashMap<String, AtomicLong> delayMap = new ConcurrentHashMap<>();
```

Comment:
Local variables inside the method do not need to use ConcurrentHashMap, just change it to HashMap.

Reply:
Done.